scripts/build.rs

```diff
@@ -11,8 +14,10 @@ fn main() {
 
 #[cfg(not(windows))]
 fn main() {
+    // Always rerun the build script regardless of the changes in the source code.
+    println!("cargo:rerun-if-changed=");
```

Comment:
The `rerun-if-changed` lines shouldn't be necessary in this build script, since the build script in `nu-cmd-lang` will automatically cause the Nushell executable to be recompiled.

Reply:
We'll still be guaranteed a rebuild if the `nu-cmd-lang` build script is rerun. The top-level build script should only need to be re-run if the build script itself is modified. Regardless of what we do with the `nu-cmd-lang` build script, it should be safe to revert this change and still get the desired behavior of rebuilding each time (since `nu-cmd-lang` is the thing we actually care about getting rebuilt)

Reply:
i reverted these changes.